src/IB/IBFEMethod.cpp

```diff
@@ -806,11 +821,14 @@ IBFEMethod::interpolateVelocity(const int u_data_idx,
                                                                  /*close_F*/ false);
```

Comment:
Similarly, it's dumb, but I would prefer to keep calling the linear solver (even if it takes exactly one iteration to solve) in every case to simplify control flow.